content/docs/configuration/content.md

```diff
@@ -6,58 +6,214 @@ title = "Configuration"
 
 [sidebar]
   enable = true
+  sticky = true
 +++
 
-**Redirect all requests to "www"-subdomain:**  
-*example.com -> www.example.com*
+# Redirect Types
+
+---
+
+## Supported redirect types:
+
+| Type Name | Type Description                                            |
+| --------- | ----------------------------------------------------------- |
+| www       | Redirect all requests to "www"-subdomain                    |
+| host      | Redirect to host provided in TXT record                     |
+| path      | Enable path based redirects                                 |
+| gometa    | Enable Go packages meta/vanity redirects                    |
+| gomods    | Enable serving and caching Go modules                       |
+| dockerv2  | Redirect to docker registry or image provided in TXT record |
+| proxy     | Proxy the request to endpoint provided in TXT record        |
+
+---
+
+## Enable type
+
 ```
 txtdirect {
   enable www
 }
 ```
 
-**Redirect to host provided in TXT record:**  
-*Fallback: Return 404 on empty record*
+## Enable multiple types
+
 ```
 txtdirect {
-  enable host
+  enable host path gomods dockerv2
 }
 ```
 
-**Redirect to host provided in TXT record:**  
-*Fallback: Redirect to "www"-subdomain on empty record*
+## Disable type
+
 ```
 txtdirect {
-  enable host www
+  disable www
 }
 ```
 
-**Redirect to host provided in TXT record:**  
-*Fallback: Redirect to example.com on empty record*
+# Fallback
+
+---
+
+## Flow
+
+Step-by-Step fallback flow:
+
+- Specific record's `to=` field
+- Specific record's `website=` field
+- Specific record's `root=` field
+- Second to last record's `website=` field
+- Second to last record's `root=` field
+- Second to last record's `to=` field
+- Request's host "www"-subdomain
+- Global config's `redirect` field
+- 404 Status code
+
+_hint: **Specific record** is the last TXT record found_  
```

Comment:
Why use a hint? Instead of "specific record" we could say "last found TXT record" above. Maybe both are not clear, but a hint feels like a workaround, that doesn't resolve this.

Reply:
Ideas: (happy for other suggesitons)
`- Current TXT record *to=* field`
`- Parent (type=path) TXT record *to=* field`

Reply:
Since we only support `path` as a parent record for now, Think it would be fine to use the examples you mentioned.